.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: false,
+    node: true,
+  },
+  plugins: ['node', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:node/recommended',
+    'plugin:prettier/recommended',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
```

Comment:
I think this isn't right? I think we want 2019 or above.

Reply:
12 = 2021
From their [docs](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments):

_ecmaVersion - set to 3, 5 (default), 6, 7, 8, 9, 10, 11, or 12 to specify the version of ECMAScript syntax you want to use. You can also set to 2015 (same as 6), 2016 (same as 7), 2017 (same as 8), 2018 (same as 9), 2019 (same as 10), 2020 (same as 11), or 2021 (same as 12) to use the year-based naming._

Reply:
So this is very confusing to read. I think we should use the year. I've never seen it configured this way before (and people may thing the config is broken).